30413/Alisa Dariana Roman/Lab10/p_10_min_path/main.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int M[3][4] = { {1, 1, 2, 1},
+                {1, 5, 3, 6},
+                {3, 5, 1, 2}
+              };
+
+int minM[3][4];
+
+void printMatrix(int M[3][4], int dim1, int dim2)
+{
+    int i, j;
+    for(i = 0; i < dim1; i++) {
+        for(j = 0; j < dim2; j++) {
+            printf("%5d", M[i][j]);
+        }
+        printf("\n");
+    }
+}
+
+int smallest(int x, int y, int z)
+{
+    int c = 0;
+    while (x && y && z) {
+        x--;
+        y--;
+        z--;
+        c++;
+    }
+    return c;
+}
+
```

Comment:
is time complexity of this O(n*m)? ( i calculated values for the first row and first column in main)